config/consensus.go

```diff
@@ -1202,7 +1202,7 @@ func initConsensusProtocols() {
 	vFuture := v34
 	vFuture.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
 
-	vFuture.LogicSigVersion = 8 // When moving this to a release, put a new higher LogicSigVersion here
+	vFuture.LogicSigVersion = 8 // When moving this to a release, put a new higher LogicSigVersion here + LogicVersion in data/transactions/logic/opcodes.go + docVersion in cmd/opdoc/opdoc.go.
```

Comment:
I think theres too much going on in the choices here to reduce it to this rule.  Else we would just write the code that way.